### ISSUE TITLE
Question about the Alignet Dataset?

### ISSUE THREAD

**Issue by wxl-001** [google-deepmind/alignet#6]
From the released Alignet datasets (https://storage.googleapis.com/alignet/data/release_1.1/index.html), I understand that it contains two folders: UD and Untransformed, which, as stated in the paper, correspond to two imperfect settings, in contrast to the superior AligNet/Soft-alignment setting. Because neither of these two folders contains data that can be directly used to fine-tune the student model in the way described in the paper (leveraging soft similarity targets from the teacher model, i.e., SigLIP-So400m after UD transformation).

This raises two questions:

1. Is the "ideal" training data — i.e., image triplets processed by the teacher model (SigLIP-So400m) after UD transformation, yielding (filenames: (n, 3), similarities: (n, 3)) — not publicly released? In other words, is only the input to the teacher (UD similarities) released, but not the teacher’s output (similarities for fine-tuning student model)?

2. If the answer to the first question is yes, does that mean we are expected to:
- Perform Uncertainty Distillation on SigLIP-So400m using data in the UD folder;
- Then run the distilled teacher model on the same triplets to generate new (n, 3) similarity vectors;
- Use these distilled-teacher similarities as soft targets to train the student model.

If so, why are these distilled similarities not released? Releasing them would greatly improve reproducibility instead of only releasing data of UD or untransformed.

Perhaps I misunderstood the usage of the released alignet dataset.
Thank you for clarifying — this is critical for reproducing the reported results and understanding the intended training pipeline.

**Comment by lampinen-dm** [comment]
Hi, apologies for the slow response. Sorry if the mapping between the paper and the released data are unclear. The UD training data we released are what is described in the paper as the "AligNet Data" labeled by the surrogate teacher model. The surrogate teacher model in question is adapted using the uncertainty-distillation loss, as described in the paper. These are therefore precisely the data used to train the student models. 

We named the above folder UD to contrast it with the "Untransformed" ablation case, which we refer to as "UnAligNet" in the paper: i.e., the case where we use the baseline model instead of the UD-transformed surrogate teacher. 

I hope this clarifies things, please follow up if you have further questions.
